app/components/checkbox/checkbox.tsx

```diff
@@ -5,13 +5,13 @@ type CheckboxProps = { label: React.ReactNode } & RadixCheckbox.CheckboxProps;
 
 export function Checkbox({ label, ...props }: CheckboxProps) {
   return (
-    <div className="flex items-center space-x-2 text-zinc-500">
+    <div className="flex items-center space-x-2 text-gray-600">
       <RadixCheckbox.Root {...props} className="w-4 h-4 rounded ring-1 ring-inset ring-black/10">
         <RadixCheckbox.Indicator>
           <CheckIcon />
         </RadixCheckbox.Indicator>
       </RadixCheckbox.Root>
-      <label>{label}</label>
+      <label htmlFor={props.name}>{label}</label>
```

Comment:
Look like htmlFor works on id. Maybe should be `props.id` and let the user decide the id name?